sycl/cmake/modules/FetchUnifiedRuntime.cmake

```diff
@@ -125,6 +125,9 @@ if(SYCL_UR_USE_FETCH_CONTENT)
   #    [XPTI] Use `ur.call` rather than `ur` in XPTI
   set(UNIFIED_RUNTIME_TAG d0a50523006fa6f283da6a36811081add3bb22fc)
 
+  set(UNIFIED_RUNTIME_REPO "https://github.com/yingcong-wu/unified-runtime.git")
+  set(UNIFIED_RUNTIME_TAG "yc-0903-handle-option-exception")
```

Comment:
updating the tag here is no longer necessary, https://github.com/oneapi-src/unified-runtime/pull/2054 is merged and has since been included in a tag update

Reply:
Sure, I will remove this change.